linkerd/app/outbound/src/tcp/tests.rs

```diff
@@ -37,10 +37,13 @@ async fn plaintext_tcp() {
     // ports or anything. These will just be used so that the proxy has a socket
     // address to resolve, etc.
     let target_addr = SocketAddr::new([0, 0, 0, 0].into(), 666);
+    let recv = profile::only_default();
+    let logical_addr = recv.borrow().addr.clone();
     let logical = Logical {
         orig_dst: OrigDstAddr(target_addr),
-        profile: Some(profile::only_default()),
+        profile: Some(recv),
         protocol: (),
+        logical_addr,
```

Comment:
nit/TIOLI: might be nice to have a function so we don't have to repeat this everywhere in the tests?

Reply:
Added 👍